src/main/java/core/basesyntax/db/DataBase.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataBase {
+    public static final Map<String, Integer> storage = new HashMap<>();
```

Comment:
According to the encapsulation principle, the `storage` map should not be public. Consider using a private access modifier and providing access through public methods.